### DISCUSSION TITLE
More functionality in Frigate+

### DISCUSSION THREAD

**Discussion by rayzorben** [blakeblackshear/frigate#11340] [Ideas]
It's starting to become difficult to manage images in Frigate+, so looking for the following functionality

1. the ability to just iterate through images <Previous Next> so that I can double check things, or at least the ability to go back to the previous image when you realize you missed something or mislabeled something
2. the ability to filter non verified images or verified images only
3. the ability to filter and see images with person/dog/cat etc only

**Comment by blakeblackshear** [comment]
I captured this here: https://github.com/orgs/frigate-nvr/projects/2/views/1?pane=issue&itemId=52876634
